README.md

```diff
@@ -18,6 +18,92 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #01(number) - Separate name and surname from fullname string.
+
+> yyyy-mm-dd(date) by [@0xmtn](https://github.com/0xmtn/)
+
+Sometimes, for the sake of UI, it is a neccessity to have a full control over a fullname when displaying it.
+
+```javascript
+Array.prototype.abbrFullName = function(name, surname){ 
+    return this.map(function(elem, idx, arr){
+        return idx != 0 && idx != arr.length-1 ? elem[0]+"." : 
+            (name && idx==0) || (surname && idx==arr.length-1) ? elem[0]+"." : 
+            elem;
+    }).join(" ");
+}
+```
+Examples:
+
+```javascript
+var fullname_str = "John Ronald Reuel Tolkien";
+
+var fullname_word_list = fullname_str.split(" ");
+console.log(fullname_word_list);
+// ["John", "Ronald", "Reuel", "Tolkien"]
+
+fullname_word_list.abbrFullName(false, false); //abbreviate only the middle names
```

Comment:
See the [Boolean Trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html)